website/src/routes/api/FixedSizeGrid.js

```diff
@@ -300,13 +312,25 @@ const PROPS = [
         <strong className={styles.DeprecatedProp}>
           This property has been deprecated.
         </strong>{' '}
-        Please use the <code>overscanColumnsCount</code> and{' '}
-        <code>overscanRowsCount</code> properties instead.
+        Please use the <code>overscanColumnCount</code> and{' '}
+        <code>overscanRowCount</code> properties instead.
       </p>
     ),
     name: 'overscanCount',
     type: 'number',
   },
+  {
+    description: (
+      <p>
+        <strong className={styles.DeprecatedProp}>
+          This property has been deprecated.
+        </strong>{' '}
+        Please use the <code>overscanRowCount</code> property instead.
+      </p>
+    ),
+    name: 'overscanRowsCount',
```

Comment:
I think you forgot to rename `overscanRowsCount` -> `overscanRowCount` for the pre-existing one. 😁 I'll do it!